hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/TestHddsDatanodeService.java

```diff
@@ -149,7 +150,7 @@ public void testDeletedContainersClearedOnShutdown(String schemaVersion)
     StorageVolume volume = volumeSet.getVolumesList().get(0);
 
     // Check instanceof and typecast
-    assertTrue(volume instanceof HddsVolume);
+    assertInstanceOf(HddsVolume.class, volume);
     HddsVolume hddsVolume = (HddsVolume) volume;
```

Comment:
Just a note for future patches, no need to update this patch for this now.

`assertInstanceOf` returns the object after cast, so this can be simplified to:

```
HddsVolume hddsVolume = assertInstanceOf(HddsVolume.class, volume);
```